src/backend/views/pages/form.twig

```diff
@@ -13,14 +13,18 @@
       }
     </textarea>
     <header class="writing-header">
-      <span class="writing-header__left">
-        <span>
-        New Page at the
+      <div class="writing-header__left">
```

Comment:
is this class name still actual?

Reply:
why `left`?

Reply:
Renamed to `inner-container` since `left` has no meaning now